bot/exts/easter/earth_photos.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+import discord
+import requests
+from discord.ext import commands
+
+from bot.constants import Tokens
+
+log = logging.getLogger(__name__)
+
+UnClient_id = Tokens.unsplash_key
+
+
+class EarthPhotos(commands.Cog):
+    """This cog contains the command for earth photos."""
+
+    def init(self, bot: commands.Bot) -> None:
```

Comment:
```suggestion
    def __init__(self, bot: commands.Bot) -> None:
```
`init` has no special meaning in Python. It's just a method.

`__init__` is the magic method that's called automatically on object instantiation.